### ISSUE TITLE
Jest Can't Parse theme.js Import -- Unexpected Token in JSON at Position 0

### ISSUE THREAD

**Issue by Harrison-Lara** [tinymce/tinymce#5279]
**Linked PR**
https://github.com/tinymce/tinymce/pull/5280

**What is the current behavior? Describe the bug**

My environment is using Jest, TS and the React TinyMCE editor. Using the component and silver theme, there is not an issue. When running Jest, there is an unexpected token in JSON at position 0 in silver/theme.js. The issue still persists when using the import and export loaders with webpack. I had to use a module transformer in Jest to get past the CSS issues, but its not possible to transform the theme.js file. This is a current roadblock to using TinyMCE. 

![image](https://user-images.githubusercontent.com/42361855/69446850-e6295280-0d1a-11ea-924c-0baa02398d0a.png)

I tracked the issue down to the ColorCache.ts file. 
I added a JSON.stringify to the storageString. This does not change the color functionality, editor behavior, and Jest now passes all tests and parses the silver/theme import without issues. 

![image](https://user-images.githubusercontent.com/42361855/69446444-0c022780-0d1a-11ea-8399-dc0c9a344963.png)

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

![image](https://user-images.githubusercontent.com/42361855/69447037-41f3db80-0d1b-11ea-8d47-8bcd740a7494.png)

**Fix:**

![image](https://user-images.githubusercontent.com/42361855/69447443-23daab00-0d1c-11ea-9dc3-945e61ecd890.png) 

**What is the expected behavior?**

There is no change to the functionality editor. Jest should be able to parse and handle the silver/theme.js import. 

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

Issue found in version: 5.1.2
Issue persists in Windows and Linux. Prior version behavior is unknown.

**Comment by Harrison-Lara** [comment]
I have a branch and would like to PR the fix, but currently getting a permission denied error.

**Comment by lnewson** [comment]
Thanks for trying to solve this one, but as I mentioned on the linked PR, adding the additional `JSON.stringify` does change the behaviour in an undesired way. So can I ask that you please provide a minimal/small reproduction case we could run ourselves to reproduce this? That way we can investigate more and try to find out why Jest is throwing this error and come up with another workaround that won't change the behaviour.

**Comment by Harrison-Lara** [comment]
I see that my implementation was not giving the error with the custom color picker. I created a TinyMCE demo from scratch, tested it and it does break. I have mocked my entire implementation with a small project from scratch, but cannot get the jest issue to reproduce. I'm not sure what in my jest environment is causing this different from the two projects. I'm going to continue searching for a way with Jest to exclude this import or module mock the file. My rich text editor component passes without the jest issues, but any surrounding files that share component or barrel export, then jest gets caught up in the theme import issue.
